### PR TITLE
Fixed issues with Sushanth frag data #96 and added wolf howl sound.

### PR DESCRIPTION
**Pull request by MthreeJohnnyK**
See @at7064 's [https://packofparts.slack.com/archives/C05JAHE5N8P/p1710828098456899](message). 
Also added wolf howl sound to all wolf logos. Can be a future cheer?

### CODE REVIEW

File: app/src/main/java/com/example/myapplication/SecondFragment.java [hunk 1]
```diff
@@ -78,6 +78,7 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
             animation.start();
             UIHelpers.darkMode = !UIHelpers.darkMode;
             UIHelpers.lightDark(v, UIHelpers.darkMode);
+            UIHelpers.playHowlSound(this.getContext());
```
Comment (carlosfigueira) [inline app/src/main/java/com/example/myapplication/SecondFragment.java:81]: Those four lines are the same in many fragments. Consider moving them to a method in UIHelpers:

    public static void darkModeToggle(ObjectAnimator animation, Context context) {
        animation.start();
        darkMode = !darkMode;
        lightDark(darkMode);
        playHowlSound(context);
    }

Comment (MthreeJohnnyK) [reply app/src/main/java/com/example/myapplication/SecondFragment.java:81]: Ok ;)

Comment (MthreeJohnnyK) [reply app/src/main/java/com/example/myapplication/SecondFragment.java]: Done

**Review by carlosfigueira — APPROVED** [review]
:shipit:

**Pull request merged**